solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -368,6 +370,86 @@ contract StandardPoolConverter is
+        (uint256 reserveBalance0, uint256 reserveBalance1) = transferFeesAndSyncReserveBalances(0);
+        __reserveBalancesProd = reserveBalance0 * reserveBalance1;
+    }
+
+    /**
+     * @dev transfers a portion of the accumulated conversion fees and syncs the reserve balances
+     *
+     * @param _value    amount of ether to exclude from the ether reserve balance (if relevant)
+     * @return new reserve balances
+     */
+    function transferFeesAndSyncReserveBalances(uint256 _value) internal returns (uint256, uint256) {
+        syncReserveBalances(_value);
+
+        (uint256 reserveBalance0, uint256 reserveBalance1) = reserveBalances(1, 2);
+        (uint256 currPoint, uint256 prevPoint) = networkFeePoints(reserveBalance0 * reserveBalance1, __reserveBalancesProd);
+
+        if (currPoint > prevPoint) {
+            INetworkSettings networkSettings = INetworkSettings(addressOf(NETWORK_SETTINGS));
+            (address networkFeeWallet, uint32 networkFee) = networkSettings.networkFeeParams();
+            (uint256 n, uint256 d) = networkFeeRatio(currPoint, prevPoint, networkFee);
+            uint256 fee0 = reserveBalance0.mul(n).div(d);
+            uint256 fee1 = reserveBalance1.mul(n).div(d);
+            reserveBalance0 -= fee0;
+            reserveBalance1 -= fee1;
+            setReserveBalances(1, 2, reserveBalance0, reserveBalance1);
+            safeTransfer(__reserveTokens[0], networkFeeWallet, fee0);
+            safeTransfer(__reserveTokens[1], networkFeeWallet, fee1);
+        }
+
+        return (reserveBalance0, reserveBalance1);
+    }
+
+    /**
+     * @dev returns the reserve balances of the given reserve tokens minus their corresponding fees
+     *
+     * @param _reserveTokens    reserve tokens
+     * @return reserve balances minus their corresponding fees
+     */
+    function reserveBalancesMinusFees(IERC20[] memory _reserveTokens) internal view returns (uint256[2] memory) {
+        uint256 reserveId0 = __reserveIds[_reserveTokens[0]];
+        uint256 reserveId1 = __reserveIds[_reserveTokens[1]];
+        (uint256 reserveBalance0, uint256 reserveBalance1) = reserveBalances(reserveId0, reserveId1);
+        (uint256 currPoint, uint256 prevPoint) = networkFeePoints(reserveBalance0 * reserveBalance1, __reserveBalancesProd);
+
+        if (currPoint > prevPoint) {
+            uint32 networkFee = INetworkSettings(addressOf(NETWORK_SETTINGS)).networkFee();
+            (uint256 n, uint256 d) = networkFeeRatio(currPoint, prevPoint, networkFee);
+            return [
+                reserveBalance0.sub(reserveBalance0.mul(n).div(d)),
+                reserveBalance1.sub(reserveBalance1.mul(n).div(d))
+            ];
+        }
+
+        return [reserveBalance0, reserveBalance1];
+    }
+
+    /**
+     * @dev returns the reserve balances of the given reserve token minus its corresponding fee
+     *
+     * @param _reserveToken reserve token
+     * @return reserve balance minus its corresponding fee
```

Comment:
We actually don't have a consistent convention for that throughout our code, but fixed.

Reply:
The only reason that we don't a consistent convention is since we keep postponing standardizing legacy code, but new or modified/touch code should be written according to our new conventions